tiaas.md

```diff
@@ -39,6 +39,8 @@ jobs on a training machine, and, in the event there is no more capacity, they
 will run on the main queue. If a spot on a training VM opens up first, they
 will run there rather than continuing to wait in the main queue.
 
+Before the training, the students need to create an account in [useGalaxy.eu](https://usegalaxy.eu/) and the data will stay available once the training is finished.
```

Comment:
Before users can join a training, they need to be logged into the European Galaxy server ... dot.
After a training the data will stay available if desired. However, if this is not needed we kindly ask to clean your histories.

Something like that?